gradle.properties

```diff
@@ -65,3 +65,16 @@ ws_version=6.2.1
 # android (https://developer.android.com/studio/releases/gradle-plugin)
 android_gradle_version=3.5.3
 android_stetho_version=1.5.1
+
+# Workaround for Bintray treating .sha512 files as artifacts
+# https://github.com/gradle/gradle/issues/11412
+systemProp.org.gradle.internal.publish.checksums.insecure=true
+
+kotlin.mpp.enableGranularSourceSetsMetadata=true
+kotlin.mpp.enableCompatibilityMetadataVariant=true
+
+kotlin.native.disableCompilerDaemon=true
+
+kotlin.native.distribution.type=prebuilt
+
+org.gradle.parallel=false
```

Comment:
It takes lots of time to build and publish without parallel. 
Could we fix this?

Reply:
Fixed it, thanks. That was a local change that I should not have committed.